src/PlaceProperties.ts

```diff
@@ -106,6 +120,13 @@ export const PlacePropertiesSchema = new SimpleSchema({
       componentHint: 'Address'
     }
   },
+  levels: {
+    type: LevelSchema,
+    optional: true,
+    accessibility: {
+      question: t`Describe the accessibility of the place's levels.`
```

Comment:
```suggestion
      question: t`Describe the accessibility of the place’s levels.`
```

(The docs use curly punctuation to differentiate between strings in code and quoted text. Apostrophes should be consistent with this)